src/reducers/mcode.js

```diff
@@ -18,11 +19,13 @@ export default function mcode(state = defaultState, action) {
             isSimilarPatient(treatmentDataPatient, state.similarPatientProps));
         const similarPatientTreatments = generateSimilarPatientTreatments(similarPatients);
 
+        const timescale = ['1','3','5'];
```

Comment:
However we decide that the timescales will be configurable they must be a property that can be configured in the config.js file that is located in the public folder.